http-api/src/main/java/net/runelite/http/api/item/ItemStats.java

```diff
@@ -42,10 +42,9 @@ public ItemStats subtract(ItemStats other)
 			return this;
 		}
 
-		final double newWeight = weight - other.weight;
+		final double newWeight = Math.round((weight - other.weight) * 10000) / 10000d;
```

Comment:
Why did you changed this?

Reply:
so it doesn't show double stats as `3.24522222222222` as I was experiencing

Reply:
Why not format the number to the desired number of decimals when you convert it into a string, like how `QuantityFormatter#PRECISE_DECIMAL_FORMATTER` works

Reply:
to what benefit? The value needs to be a double so it would just need to be converted back to a double

Reply:
Because you just want to display it in different format, and changing the value in core api is bad way to do it.

Reply:
to be clear it's only the new weight that has this format issue, when you subtract the other weight from it that is when the large double decimal format occurs. So it seems to me that this is the appropriate place for this given the format is from the api

Reply:
I will revert and update to use the QuantityFormatter, but not sure I see the issue here personally. I could be missing something though.